### DISCUSSION TITLE
Please tell us about using Birt in Java8.

### DISCUSSION THREAD

**Discussion by tanitanichan** [eclipse-birt/birt#1185] [Q&A] [answered]
Hello, Birt-TEAM.

Please tell us about using Birt in Java8.

(1) In the latest content of Oracle's JavaSE roadmap, the support period for Java8 has been reviewed, and a longer extended support period than Java11 and Java17 has been set, but Birt plans to provide a version that runs on Java8 in the future. is there?

(2) How can I obtain a list of unresolved bugs as of 4.8.0 for version 4.8.0, which is the latest version that can run on Java8 at the moment? (Same as 4.9.0's list of resolved bugs?) Also, will there be patches available for any critical outstanding issues?

Thanks in advance,
Regards,
Tani

**Comment by wimjongman** [comment, marked as answer]
(1) BIRT does not provide a version that runs on Java 8
(2) For the buglist please look at the issues. There will be no patches on critical issues for old versions unless you want to pay for them.

Issues for BIRT 4.9 to 4.12: https://github.com/eclipse/birt/milestones?state=closed
Open issues: https://github.com/eclipse/birt/milestones

**Reply by tanitanichan** [reply]
Hi, wimjongman.

Thank you for the immediate reply.
I heard what I wanted to hear, so I'm closing this case.
